Templates/sample_inventory/README.md

```diff
@@ -12,6 +12,16 @@ hesitate to contact the Data Curation team at
 [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
 so we can assist you.
 
+# Sample Inventory Cheat Sheet
+The following is a short list of best practices when filling out this template:
+* Use `Sample ID` values that match the IDs submitted to data archives
+* `Sample Type` refers to the type of biological entity that `Sample ID` identifies
+* `Subspecimen Type` is different from `Sample Type` refers to the type of 
```

Comment:
Wouldn't this be a subsample? Is a slice a biological entity? Is a library a biological entity?